src/Spout/Reader/XLSX/Helper/CellValueFormatter.php

```diff
@@ -189,7 +189,17 @@ protected function formatExcelTimestampValue($nodeValue)
 
         // The value 1.0 represents 1900-01-01. Numbers below 1.0 are not valid Excel dates.
         if ($nodeValue < 1.0) {
-            return null;
+            // treat this value as time-only cell
+            $time = round($nodeValue * 86400);
```

Comment:
can you replace all numbers with constants defined at the top of the file?

NUM_SECONDS_IN_ONE_DAY, NUM_SECONDS_IN_ONE_HOUR, NUM_SECONDS_IN_ONE_MINUTE